app/controllers/api/deploy_groups_controller.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+class Api::DeployGroupsController < Api::BaseController
+  before_action :ensure_enabled
+
+  def index
+    scope = current_project.present? ? stage.deploy_groups : DeployGroup.all
+    render json: paginate(scope.sort_by(&:natural_order))
```

Comment:
I see. Hmm. Yeah, maybe skip the paginate anyways? I think we only need it for things like deploys which are definitely more than 1000?